tests/cypress/latest/support/commands.ts

```diff
@@ -572,20 +572,12 @@ Cypress.Commands.add('checkChart', (operation, chartName, namespace, version, qu
   }
 
   cy.clickButton(operation);
+  // This is 1s more than the time required for the installation tabpanel to appear;
+  // or in case of Turtles, Rancher pod restarts, so this is enough time to start restarting Rancher
+  cy.wait(8000);
```

Comment:
```suggestion
  cy.wait(10000);
```
To reduce more flakes